frontend/src/core/loaders/components/SkeletonLoader.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import './SkeletonLoader.css';
+
+let list;
+
+export default function SkeletonLoader (props) {
+    { list = props.entities.length === 0 ? [...Array(30).keys()] : [...Array(2).keys()] }
+    return <ul className={`skeleton-loader entities ${props.entities.length === 0 ? 'entities' : 'loading'}`}>
```

Comment:
You are appending the `entities` class twice. It should be `null : 'scroll'`.

Also: spaces around content within `{}` and `${}`.